<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QMainWindow" name="Manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="panel" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget_Pages"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>396</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuService">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuToolbars">
    <property name="toolTip">
     <string>Toolbar</string>
    </property>
    <property name="statusTip">
     <string>Toolbar</string>
    </property>
    <property name="whatsThis">
     <string>Toolbar</string>
    </property>
    <property name="title">
     <string>Toolbars</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuToolbars"/>
   <addaction name="menuService"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockMain">
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Main</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="3" column="0">
      <widget class="QToolButton" name="tButtonQuit">
       <property name="toolTip">
        <string>Quit</string>
       </property>
       <property name="statusTip">
        <string>Quit</string>
       </property>
       <property name="whatsThis">
        <string>Quit</string>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
       <property name="icon">
        <iconset resource="Manager.qrc">
         <normaloff>:/quit</normaloff>:/quit</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QToolButton" name="tButtonSettings">
       <property name="toolTip">
        <string>Settings</string>
       </property>
       <property name="statusTip">
        <string>Settings</string>
       </property>
       <property name="whatsThis">
        <string>Settings</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="icon">
        <iconset resource="Manager.qrc">
         <normaloff>:/settings</normaloff>:/settings</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QToolButton" name="tButtonAbout">
       <property name="toolTip">
        <string>About</string>
       </property>
       <property name="statusTip">
        <string>About</string>
       </property>
       <property name="whatsThis">
        <string>About</string>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset resource="Manager.qrc">
         <normaloff>:/about</normaloff>:/about</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/quit</normaloff>:/quit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit</string>
   </property>
   <property name="whatsThis">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/about</normaloff>:/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Manager.qrc">
     <normaloff>:/settings</normaloff>:/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Settings</string>
   </property>
   <property name="whatsThis">
    <string>Settings</string>
   </property>
  </action>
   <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="statusTip">
    <string>Documentation</string>
   </property>
   <property name="whatsThis">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Manager.qrc"/>
  <include location="../Note/Note.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Manager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
